addons/stock_account/models/stock_move.py

```diff
@@ -555,15 +555,23 @@ def _account_analytic_entry_move(self):
             self.env['stock.move'].browse(
                 move_id).analytic_account_line_id = analytic_line
 
+    @api.model
```

Comment:
```suggestion
```
As the method uses the content of `self`, `@api.model` doesn't apply here. More info in our [documentation](https://www.odoo.com/documentation/16.0/developer/reference/backend/orm.html#odoo.api.model).

Reply:
I see. Thanks for the pointer.